qa/standalone/ceph-helpers.sh

```diff
@@ -381,7 +388,7 @@ function test_kill_daemons() {
 # @param ... can be any option valid for ceph-mon
 # @return 0 on success, 1 on error
 #
-function run_mon_no_pool() {
+function run_mon() {
```

Comment:
this change breaks `ceph-disk/tests/ceph-disk.sh`. because it expects that `run_mon()` create rbd pool for it. see `test_activate_dir()` and `read_write()`